### ISSUE TITLE
Feature Request: Enter is not creating the new brother card in flow

### ISSUE THREAD

**Issue by Gourav1910** [Quorafind/Obsidian-Canvas-MindMap#20]
### Feature Requested

Hi, your plugin is really amazing but there is a little problem. Whenever we create a card and start typing in it then if we press enter then it just change the line inside the card only just like any text editor. It does not create a new card. For creating a new brother card we first need to click somewhere else to come out of that card's editing mode and then we again click on it and then we press enter to create a new brother card. This breaks the whole flow of creating new cards fast. So please make some changes to that enter key. You can change it to "clt + enter" or something. 

### Checklist

- [X] The feature would be useful to more users than just me.

**Comment by Quorafind** [comment]
Sure, I can make some changes to this.

**Comment by salievi** [comment]
Second this. On my system at least, there seems to be no way to create a brother card with this plugin, which takes away half the functionality.

**Comment by An-qi-Zhang** [comment]
Agree! Enter does not work consistently for me. Hotkeys like "Ctrl + Enter" probably work better.

**Comment by duckfarts** [comment]
I found that the quick way to deal with this is that when you're editing a card, press Esc to exit the card, then you can press Enter to make a sibling card.

I agree that making a hotkey for it would be best so you can set it to whatever you want. I don't think it's a good idea to make Enter create a new card when you're editing a card, because it would then prevent you from making line breaks (otherwise, you could make it a toggleable option).

**Comment by Quorafind** [comment]
Same #22

**Comment by Gourav1910** [comment]
ESC is taking out us from card's editing mode but not doing anything after hitting enter. Even though the card is still selected.

**Comment by janhink** [comment]
Hello, I'm having the same issue. ESC ends the edit mode but hitting Enter afterwards does nothing. The only solution is to click outside and then select the card again, or do the same by traversing to another node with keyboard, then returning back and then hitting enter.
